pyworkflow/plugin.py

```diff
@@ -40,7 +40,7 @@
 import pyworkflow as pw
 import pyworkflow.utils as pwutils
 import pyworkflow.object as pwobj
-
+from .constants import *
```

Comment:
Can we import just the constants explicitly?

Reply:
@pconesa I think that we agreed that we will expose all constants at the modules level (e.g pw here or plugins such as xmipp, relion, etc).
I think in this case (only importing constants) the import * should be fine here, to avoid explicitly adding all constants.

Reply:
Ah, sorry! I didn't notice that this is in the plugin.py submodule...then I completely agree with you to import only the required ones.

Reply:
Ok, I thought we were doing it the other way around.